### PR TITLE
implemented method to retrieve path and urls

### PR DESCRIPTION
**Pull request by giovannialbero1992**
# Description

Started to add path and urls mentioned in the issue below

Related to issue #298 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Comment by giovannialbero1992** [comment]
Hi @pieroit, I didn't understand what you meant about `staticURL` .
Can you explain a little bit pls? Thanks :smiley:

File: core/cat/looking_glass/cheshire_cat.py [hunk 1]
```diff
@@ -174,6 +175,15 @@ def store_new_message_in_working_memory(self, user_message_json):
         self.working_memory["user_message_json"]["prompt_settings"] = prompt_settings
 
 
+    def get_base_url(self):
+        return os.getenv("CORE_HOST")
```
Comment (pieroit) [inline core/cat/looking_glass/cheshire_cat.py:179]: Please also use `CORE_SECURE_PROTOCOLS` and `CORE_PORT`
Example [here](https://github.com/cheshire-cat-ai/core/blob/main/core/cat/log.py#L167)

File: core/tests/looking_glass/test_cheshire_cat.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+from cat.looking_glass.cheshire_cat import CheshireCat
+
+def test_get_base_url():
+    cat = CheshireCat()
+    assert cat.get_base_url() == "localhost"
```
Comment (pieroit) [inline core/tests/looking_glass/test_cheshire_cat.py:5]: should be `http://localhost:1865`

File: core/tests/looking_glass/test_cheshire_cat.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+def test_base_path():
+    cat = CheshireCat()
+    assert cat.get_base_path() == "/app"
```
Comment (pieroit) [inline core/tests/looking_glass/test_cheshire_cat.py:9]: should be `/app/cat/`

File: core/tests/looking_glass/test_cheshire_cat.py [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+def test_plugin_path():
+    cat = CheshireCat()
+    assert cat.get_plugin_path() == "/app/plugins"
```
Comment (pieroit) [inline core/tests/looking_glass/test_cheshire_cat.py:13]: should be `/app/cat/plugins`

Comment (pieroit) [reply core/tests/looking_glass/test_cheshire_cat.py:13]: static_path should be `/app/cat/static` and static_url `http://localhost:1865/static/` 

Very useful for plugins to store and give URLs of static files

File: core/cat/looking_glass/cheshire_cat.py (hunk 1 above)
Comment (giovannialbero1992) [reply core/cat/looking_glass/cheshire_cat.py]: fixed

File: core/tests/looking_glass/test_cheshire_cat.py (hunk 2 above)
Comment (giovannialbero1992) [reply core/tests/looking_glass/test_cheshire_cat.py]: fixed

File: core/tests/looking_glass/test_cheshire_cat.py (hunk 3 above)
Comment (giovannialbero1992) [reply core/tests/looking_glass/test_cheshire_cat.py]: fixed

File: core/tests/looking_glass/test_cheshire_cat.py (hunk 4 above)
Comment (giovannialbero1992) [reply core/tests/looking_glass/test_cheshire_cat.py]: Added even the static path and fixed the path resolution

**Comment by pieroit** [comment]
Thanks adn welcome. :)

**Pull request merged**